src/licensedcode/data/rules/gpl-2.0_and_gpl-1.0-plus_and_gpl-2.0-plus_and_lgpl-2.0_and_lgpl-2.0-plus_and_lgpl-2.1_and_lgpl-2.1-plus_1.RULE

```diff
@@ -0,0 +1,17 @@
+---
+license_expression: gpl-2.0 AND gpl-1.0-plus AND gpl-2.0-plus AND lgpl-2.0 AND lgpl-2.0-plus
+    AND lgpl-2.1 AND lgpl-2.1-plus
+is_license_notice: yes
+minimum_coverage: 90
+---
+
+The license described in the COPYING file applies to the kernel source
```

Comment:
Are you sure of the license? the COPYING file in the kernel source would be GPL-2.0-only with Linux-syscall-notes
```suggestion
The license described in the {{COPYING file applies to the kernel source}}
```

Reply:
lower score otherwise.